week1/homework/src/server.js

```diff
@@ -1,20 +1,42 @@
 'use strict';
 
-const http = require('http');
+{
+  const http = require('http');
 
-/* `createServer` MUST return an instance of `http.Server` otherwise the tests
- * will fail.
- */
-function createServer(port) {
-  let state = 10;
+  const postData = (status, response, data) => {
```

Comment:
nice that you separated this in a function 👍

Reply:
Hi @gajduk !
Thank you very much for your efforts to review and making a note.